packages/web/docs/src/components/cli-errors.tsx

```diff
@@ -0,0 +1,409 @@
+/* eslint-disable tailwindcss/no-custom-classname */
+import { ReactElement } from 'react';
+import type { GetStaticProps } from 'next';
+import { Callout, Code } from '@theguild/components';
+
+type CLIError = {
+  code: string;
+  title: string;
+  example: string;
+  exampleOutput: string;
+  fix: string;
+};
+
+export function ErrorDetails(props: CLIError): ReactElement {
+  return (
+    <>
+      <h3
+        id={`errors-${props.code}`}
+        className="_font-semibold _tracking-tight _text-slate-900 _mt-8 _text-2xl dark:_text-slate-100"
+      >
+        {props.code} "{props.title}"{' '}
+        <a
+          href={`#errors-${props.code}`}
+          className="nextra-focus subheading-anchor"
+          aria-label="Permalink for this error code"
+        />
+      </h3>
+      <h4 className="_font-semibold _tracking-tight _text-slate-900 _mt-8 _text-xl dark:_text-slate-100">
```

Comment:
I'm not a big fan of having to replicate all these classes manually that the mdx apply normally for us, but I couldn't find a better option. Open to suggestions.